File: app/src/main/java/com/alvinhkh/buseta/kmb/model/KmbRoute.java
package com.alvinhkh.buseta.kmb.model;

import com.google.gson.annotations.SerializedName;

public class KmbRoute {
  @SerializedName("Destination_ENG")
  public String destinationEn;

  @SerializedName("Origin_ENG")
  public String originEn;

  @SerializedName("Origin_CHI")
  public String originTc;

  @SerializedName("To_saturday")
  public String toSaturday;

  @SerializedName("From_saturday")
  public String fromSaturday;

  @SerializedName("Desc_CHI")
  public String descTc;

  @SerializedName("Desc_ENG")
  public String descEn;

  @SerializedName("ServiceType")
  public String serviceType;

  @SerializedName("Route")
  public String route;

  @SerializedName("Destination_CHI")
  public String destinationTc;

  @SerializedName("Bound")
  public String bound;

  @SerializedName("From_weekday")
  public String fromWeekday;

  @SerializedName("From_holiday")
  public String fromHoliday;

  @SerializedName("To_weekday")
  public String toWeekday;

  @SerializedName("To_holiday")
  public String toHoliday;

}

File: app/src/main/java/com/alvinhkh/buseta/kmb/model/KmbRoute.kt
package com.alvinhkh.buseta.kmb.model

import com.google.gson.annotations.SerializedName

data class KmbRoute(
        @SerializedName("Destination_ENG")
        var destinationEn: String = "",

        @SerializedName("Origin_ENG")
        var originEn: String = "",

        @SerializedName("Origin_CHI")
        var originTc: String = "",

        @SerializedName("To_saturday")
        var toSaturday: String = "",

        @SerializedName("From_saturday")
        var fromSaturday: String = "",

        @SerializedName("Desc_CHI")
        var descTc: String = "",

        @SerializedName("Desc_ENG")
        var descEn: String = "",

        @SerializedName("ServiceType")
        var serviceType: String = "",

        @SerializedName("Route")
        var route: String = "",

        @SerializedName("Destination_CHI")
        var destinationTc: String = "",

        @SerializedName("Bound")
        var bound: String = "",

        @SerializedName("From_weekday")
        var fromWeekday: String = "",

        @SerializedName("From_holiday")
        var fromHoliday: String = "",

        @SerializedName("To_weekday")
        var toWeekday: String = "",

        @SerializedName("To_holiday")
        var toHoliday: String = ""
)
